freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -251,12 +258,24 @@ def handle(self, event):
                 # to rebuild list
                 to_rebuild_digests.add(digest)
 
-        # Skip that event because we can't proceed with processing it.
-        # TODO
-        # Now when we have bundle images' nvrs we can procceed with rebuilding it
-        msg = f"Skipping the rebuild of {len(to_rebuild_digests)} bundle images " \
-              "due to being blocked on further implementation for now."
-        db_event.transition(EventState.SKIPPED, msg)
+        builds = self._prepare_builds(db_event, bundles_by_digest,
+                                      to_rebuild_digests)
+
+        # Reset context to db_event.
+        self.set_context(db_event)
+
+        if not builds:
+            msg = f"No bundle images to rebuild for advisory {event.advisory.name}"
+            self.log_info(msg)
+            db_event.transition(EventState.SKIPPED, msg)
+            db.session.commit()
+            return []
+
+        self.start_to_build_images(builds)
+        msg = f"Advisory {db_event.search_key}: Rebuilding " \
+              f"{len(db_event.builds.all())} container images."
```

Comment:
This will make it clearer when reading the logs.
```suggestion
              f"{len(db_event.builds.all())} bundle images."
```